### ISSUE TITLE
Link on honeyswap pool page to view account analytics goes to wrong URL

### ISSUE THREAD

**Issue by divine-comedian** [1Hive/honeyswap#117]
Hi there! Using the honeyswap app on MATIC
brave browser w/ clean cache and cookies

I'm on the pool page, viewing my liquidity pools, I click the link saying `account analytics and accrued fees`

![image.png](https://images.zenhubusercontent.com/5fff1c86d883b3d9a858ddbc/1720f875-0955-4143-9b53-e7bd44bc7639)

it takes me to the honeyswap analytics homepage, but from what I see in the URL and the text the intention should take me to my account address analytics..
tl;dr 

**it directs here**
![image.png](https://images.zenhubusercontent.com/5fff1c86d883b3d9a858ddbc/7e3b830d-18c9-43be-8c86-1119d58bc737)

**it should direct here**
![image.png](https://images.zenhubusercontent.com/5fff1c86d883b3d9a858ddbc/c8cf2dbf-aa26-48b3-862d-6bd3d0cc4737)
